test/InterversionTests/SmugglerTests.cs

```diff
@@ -22,512 +11,134 @@ namespace InterversionTests
 {
     public class SmugglerTests : InterversionTestBase
     {
+        //TODO Need to be changed to version with relevant fix
+        const string Server42Version = "4.2.101";
```

Comment:
Maybe worth getting the latest dynamically?

Reply:
Probably best to stay on a fixed version, I think